app/identifiers/register/establishers/individual/PreviousPostCodeLookupId.scala

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package identifiers.register.establishers.individual
+
+import identifiers._
+import models.addresslookup.Address
+
+case object PreviousPostCodeLookupId extends TypedIdentifier[Seq[Address]] {
```

Comment:
In order for routing to work properly the `PreviousPostcodeLookupId` will need to be a case class which can accept an `id` otherwise we wouldn't know which index to send it to on the next page.